php/php-xhprof/Portfile

```diff
@@ -1,47 +1,52 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem              1.0
+PortGroup               github 1.0
 PortGroup               php 1.1
 
 name                    php-xhprof
-version                 0.9.4
+version                 2.3.7
 categories-append       devel
 platforms               darwin
-maintainers             ryandesign openmaintainer
+maintainers             {ryandesign @ryandesign} openmaintainer
 license                 Apache-2
 
-php.branches            5.3 5.4 5.5 5.6
+php.branches            7.0 7.1 7.2 7.3 7.4 8.0 8.1
```

Comment:
Instead of removing support for php 5.x, I would keep the old version for php 5.x and the new version for php >= 7, as I do in many other php module ports.

Reply:
Is this something that I can try here, or do you prefer to do it yourself. I'm happy both ways (though, if you can kindly point me to some example package doing those "doubles" that would be highly appreciated).

Ciao :-)

Reply:
Added my first attempt of multi-version in a second commit. Crossing fingers! :-)